.changelog/9ebe24c4791541e0840da49eab6f9d97.json

```diff
@@ -0,0 +1,11 @@
+{
+    "id": "9ebe24c4-7915-41e0-840d-a49eab6f9d97",
+    "type": "feature",
+    "description": "This feature adds new config fields to checksum-supported operations so user can switch on/off checksum workflow with those fields. The field RequestChecksumCalculation controls the request's checksum calculation and the field ResponseChecksumValidation controls the response's checksum validation. This feature also replaces the default MD5 checksum algorithm of checksum calculation with CRC32.",
+    "modules": [
+        ".",
+        "config",
```

Comment:
It needs to annotate every module that changed (otherwise the module will release with "no release notes available" which has confused people in the past).